zerver/views/report.py

```diff
@@ -21,12 +21,12 @@ def get_attr(csp_report_attr: str) -> str:
 
     logging.warning(
         "CSP violation in document('%s'). "
-        "blocked URI('%s'), original policy('%s'), "
+        "blocked URL('%s'), original policy('%s'), "
         "violated directive('%s'), effective directive('%s'), "
         "disposition('%s'), referrer('%s'), "
         "status code('%s'), script sample('%s')",
-        get_attr("document-uri"),
-        get_attr("blocked-uri"),
+        get_attr("document-l"),
+        get_attr("blocked-url"),
```

Comment:
These seem like mistakes? I don't see how they're related to the rest of the commit.

You should always self-review your work before asking others to review it. Please go through our [guide on submitting a pull request](https://zulip.readthedocs.io/en/latest/contributing/reviewable-prs.html) to learn how to present your proposed changes to Zulip, so that your work can be reviewed.